src/openrct2/ride/TrackDesignRepository.cpp

```diff
@@ -165,6 +189,33 @@ class TrackDesignRepository final : public ITrackDesignRepository
         return refs.size();
     }
 
+    size_t GetItemsForRideGroup(track_design_file_ref **outRefs, uint8 rideType, ride_group * rideGroup) const override
+    {
+        std::vector<track_design_file_ref> refs;
+        IObjectRepository * repo = GetObjectRepository();
+
+        for (const auto item : _items)
+        {
+            if (item.RideType != rideType)
```

Comment:
You're right, and not just for `if`s. But now we discussed this, I'll change it to use braces and put them on their own lines.